internal/service/kafka/kafka_topic_availability.go

```diff
@@ -129,29 +120,15 @@ func (w *KafkaTopicAvailabilityWaiter) refresh() error {
 	return nil
 }
 
-func (w *KafkaTopicAvailabilityWaiter) v1Refresh(queue []string) error {
-	log.Printf("[DEBUG] Kafka Topic V2 endpoit is not available, using v1!")
-	for _, t := range queue {
-		topic, err := w.Client.KafkaTopics.Get(w.Project, w.ServiceName, t)
-		if err != nil {
-			return err
-		}
-
-		cache.GetTopicCache().StoreByProjectAndServiceName(w.Project, w.ServiceName, []*aiven.KafkaTopic{topic})
-	}
-	return nil
-}
-
 // Conf sets up the configuration to refresh.
 func (w *KafkaTopicAvailabilityWaiter) Conf(timeout time.Duration) *resource.StateChangeConf {
 	log.Printf("[DEBUG] Kafka Topic availability waiter timeout %.0f minutes", timeout.Minutes())
 
 	return &resource.StateChangeConf{
-		Pending:        []string{"CONFIGURING"},
-		Target:         []string{"ACTIVE"},
-		Refresh:        w.RefreshFunc(),
-		Timeout:        timeout,
-		PollInterval:   30 * time.Second,
-		NotFoundChecks: 50,
+		Pending:      []string{"CONFIGURING"},
```

Comment:
For better error messages ( it currently says still "CONFIGURING" when timing out), can we consider "PENDING" ?

Reply:
We have the following Kafka Topic States:
```
@enum.unique
class KafkaTopicState(str, enum.Enum):
    active = "ACTIVE"
    configuring = "CONFIGURING"
    deleting = "DELETING"
``` 
so in our case it's either `CONFIGURING` or `ACTIVE`

Reply:
ah ok! sorry i thought this was our tf waiter state